src/main/interface/cli.c

```diff
@@ -4504,6 +4522,7 @@ const clicmd_t cmdTable[] = {
     CLI_COMMAND_DEF("msc", "switch into msc mode", NULL, cliMsc),
 #endif
     CLI_COMMAND_DEF("name", "name of craft", NULL, cliName),
+    CLI_COMMAND_DEF("display_name", "display name of craft", NULL, cliDisplayName),
```

Comment:
You need to add logic to `printConfig()` to include outputting this new element.  Otherwise the `display_name` won't output in dump/diff.